setup.py

```diff
@@ -61,9 +61,9 @@
     "Programming Language :: Python",
     # Make sure to specify here the versions of Python supported
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.6",
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",

```

Comment:
@rmoyard this adds the Python 3.9 support tag

Reply:
Thanks!